RadarSDKTests/RadarSDKTestsSwift.swift

```diff
@@ -0,0 +1,39 @@
+//
+//  RadarSDKTests.swift
+//  RadarSDKTests
+//
+//  Copyright © 2024 Radar Labs, Inc. All rights reserved.
+//
+
+import Foundation
+import Testing
+
+@Suite("SDK Tests")
+struct RadarSDKTestsSwift {
```

Comment:
could add a test for starting a tag and not ending it, and a test for trying to end a tag that wasn't started